src/CircleBadge.tsx

```diff
@@ -32,7 +32,7 @@ const CircleBadge = styled.div<StyledCircleBadgeProps>`
   justify-content: center;
   background-color: ${get('colors.bg.canvas')};
   border-radius: 50%;
-  box-shadow: ${get('shadows.medium')};
+  box-shadow: ${get('shadows.shadow.medium')};
```

Comment:
this was a common mistake I found - we need an extra key here to make this work. I wonder if now that we are putting shadows in their own object, this extra namespace becomes superfluous? Might be another upstream change to think about, because it is a bit awkward.

Reply:
Agreed. It's worth rethinking the naming of these in Primer Primitives